src/ontology/uberon-edit.obo

```diff
@@ -224139,6 +224139,16 @@ is_a: UBERON:0006983 ! anatomical point
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
 relationship: part_of UBERON:0001697 ! orbit of skull
 
+[Term]
+id: UBERON:8500005
+name: facial tubercle
+def: "The facial tuberosity (tuber faciale) is a prominence located on the facial surface of the maxilla; it finishes the facial crest and varies much according to the species." [https://www.imaios.com/en/vet-anatomy/anatomical-structure/facial-tuberosity-11073890408]
+synonym: "tuber faciale" EXACT []
+intersection_of: UBERON:0005813 ! tubercle
+intersection_of: part_of UBERON:0002397 ! maxilla
+property_value: curator_notes "it is well developed in equids; it is above the first molar tooth in ruminants; it is above the second molar tooth in sheep" xsd:string
+property_value: dc-contributor "https://orcid.org/0000-0001-8415-4848" xsd:string
```

Comment:
This annotation should be an IRI, not a xsd:string:

```
property_value: dc-contributor https://orcid.org/0000-0001-8415-4848
```

To do that in Protégé: when adding the `dc-contributor` annotation, instead of entering the ORCID as a literal, go to the `Entity IRI` tab and start entering directly the name of the contributor – as long as the contributor is in the “orcidio” ontology Protégé should recognise the name and insert the proper IRI (see screenshot below).

![Screenshot_20250506_221039](https://github.com/user-attachments/assets/1262a687-cf7f-4c5f-a1af-3bb51fead1d8)

Reply:
The name is not showing up for me in my protege.

Reply:
That’s weird.

But then you can also write/paste the IRI directly in the _IRI Editor_ tab:
![Screenshot 2025-05-08 at 11 37 04](https://github.com/user-attachments/assets/590d5f65-a30c-4b0c-b9c9-ba69429f6ee0)